projects/packages/status/src/class-host.php

```diff
@@ -142,9 +142,13 @@ public function get_source_query() {
 	 * @return string
```

Comment:
Since we're at it, should we `string` this to `array`?

Reply:
Good point. Done in 17671f27502dc16b0223e92e029f6b95e27a090c